src/app/frontend/replicationcontrollerlist/replicationcontrollercard.html

```diff
@@ -131,3 +131,4 @@
     </div>
   </md-card-content>
 </md-card>
+<kd-expand-parent></kd-expand-parent>
```

Comment:
Can you just style the md-card to clear:both instead of introducing a new common component directive? This would be a much easier and lightweight :)